letsdrill.rb

```diff
@@ -1,12 +1,24 @@
 def get_letter_grade(integer)
  
-  #Put your code here!
+  case
+    when integer > 90
+      return "A"
+    when integer > 80
+      return "B"
+    when integer > 70
+      return "C"
+    when integer > 60
+      return "D"
+    else
+      return "F"
+  end
```

Comment:
In ruby, it's standard to omit `return` unless it's needed for an early return or something.